modules/oe_webtools_etrans/tests/src/FunctionalJavascript/UnifiedEtransBlockTest.php

```diff
@@ -0,0 +1,131 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Drupal\Tests\oe_webtools_etrans\FunctionalJavascript;
+
+use Drupal\FunctionalJavascriptTests\WebDriverTestBase;
+use Drupal\language\Entity\ConfigurableLanguage;
+
+/**
+ * Tests the unified translations via webtool.
+ */
+class UnifiedEtransBlockTest extends WebDriverTestBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  protected $defaultTheme = 'stark';
+
+  /**
+   * {@inheritdoc}
+   */
+  protected static $modules = [
+    'block',
```

Comment:
enable page_cache and dynamic_page_cache to be sure caching is enabled

Reply:
Updated.